dask_cudf/io/tests/test_csv.py

```diff
@@ -27,4 +28,27 @@ def test_read_csv_w_bytes(tmp_path):
 
     result = df2.compute().to_pandas()
     expected = df.compute()
+    assert df2.npartitions is 3
     dd.assert_eq(result, expected, check_index=False)
+
+def test_read_csv_compression(tmp_path):
+    df = dask.datasets.timeseries(dtypes={"x": int, "y": int}, freq="120s").reset_index(
+        drop=True
+    )
+    df = pd.DataFrame(dict(x=np.arange(20), y=np.arange(20)))
+    ddf = dd.from_pandas(df, npartitions=1)
+    df.to_csv(tmp_path / "data.csv.gz", index=False, compression='gzip')
+
+    with pytest.warns(UserWarning) as w:
+        df2 = dask_cudf.read_csv(tmp_path / "*.csv.gz", chunksize="50 B", compression='gzip')
+
+    assert len(w) == 1
+    msg = str(w[0].message)
+    assert 'gzip' in msg
+
+    df2 = dask_cudf.read_csv(tmp_path / "*.csv.gz", chunksize=None, compression='gzip')
```

Comment:
```suggestion
    df2 = dask_cudf.read_csv(tmp_path / "*.csv.gz", compression='gzip')
```

Just to make sure that the fully naive thing works.

Reply:
The naive thing won't work here but I can adjust the exception check to make it work

Reply:
See recent comments around assigning `chunksize=None` if compression

Reply:
On second thought I'm not sure we should handle this case.  I think the user should have to declare the chunksize is `None`.  I suppose if compression is set we can warn and set the chunksize to `None`.   Thoughts?

Reply:
warn-and-set is the current behavior in dask.dataframe